src/azul/collections.py

```diff
@@ -270,6 +281,10 @@ def adict(seq: Union[Mapping[K, V], Iterable[tuple[K, V]]] = None,
     return kwargs if seq is None else dict(seq, **kwargs)
 
 
+def athing(cls: type, *args):
```

Comment:
Consider adding a docstring and/or doctest for this function.

Reply:
I decided to make a private function instead. It's meant just to reduce repetition among the other functions in this module, all of which are covered by docstrings, and I don't think it's a good idea for it to be used generally because I don't know how to write a satisfactory type annotation for it.